src/js/components/Tree/doc.js

```diff
@@ -0,0 +1,52 @@
+import { describe, PropTypes } from 'react-desc';
+
+import { getAvailableAtBadge } from '../../utils/mixins';
+
+export const doc = Tree => {
+  const DocumentedTree = describe(Tree)
+    .availableAt(getAvailableAtBadge('Tree', 'Controls'))
+    .description(`Hierarchical list structure.`)
+    .usage(
+      `import { Tree } from 'grommet';
+<Tree />`,
+    );
+
+  DocumentedTree.propTypes = {
+    data: PropTypes.arrayOf(PropTypes.object).description(
+      `The data structure provided to the tree, for nested behavior 
+      an object should include a children key.`,
```

Comment:
I think we also expect `name` currently. Although, see my other comment about `labelKey`.
Similarly, I also think we should support `childrenKey`. That would allow the caller to pass any object structure, using their existing data.